src/actions/ScanActions.tsx

```diff
@@ -298,7 +298,8 @@ async function sweepPrivateKeys(state: RootState, account: EdgeAccount, navigati
 
     // Check for a $50 maximum sweep for light accounts:
     const sendNativeAmount = abs(unsignedTx.nativeAmount)
-    const sendExchangeAmount = await wallet.nativeToDenomination(sendNativeAmount, wallet.currencyInfo.currencyCode)
+    const multiplier = getCurrencyCodeMultiplier(wallet.currencyConfig, wallet.currencyInfo.currencyCode)
+    const sendExchangeAmount = div(sendNativeAmount, multiplier, multiplier.length)
```

Comment:
Is this `multiplier.length` or `multiplier.length-1`? What happens if we do too much precision (more decimals then expected?)?

Reply:
Same implementation is okay by me.